swagger/swagger-generator/generator-springmvc/src/main/java/io/servicecomb/swagger/generator/springmvc/processor/annotation/ApiParamAnnotationProcessor.java

```diff
@@ -0,0 +1,25 @@
+package io.servicecomb.swagger.generator.springmvc.processor.annotation;
+
+
+import javax.ws.rs.QueryParam;
+
+import io.servicecomb.swagger.generator.core.processor.parameter.AbstractParameterProcessor;
+import io.swagger.annotations.ApiParam;
+import io.swagger.models.parameters.QueryParameter;
+
+public class ApiParamAnnotationProcessor extends AbstractParameterProcessor<QueryParameter> {
```

Comment:
ApiParam: "Adds additional meta-data for operation parameters"
it will not create a new parameter,and not always bind to queryParameter